extensions/wikia/Thumbnails/templates/Thumbnail_video.mustache

```diff
@@ -8,7 +8,7 @@
 	{{>Thumbnail_imgTag}}
 	{{#noscript}}<noscript>{{{noscript}}}</noscript>{{/noscript}}
 
-	{{#duration}}<span class="duration" itemprop="duration">{{duration}}</span>{{/duration}}
```

Comment:
That will remove duration from video metadata. Is it ok?

Reply:
currently https://search.google.com/structured-data/testing-tool#url=http%3A%2F%2Frybatest2.wikia.com%2Fwiki%2FTesting_video does not detect any of included videos, so I think it is ok. This tag didn't conform to schema.org spec https://developers.google.com/webmasters/videosearch/schema

Reply:
We could add such meta tags to every video, but I think we should consult it with stanley